### ISSUE TITLE
ec key and ec signed certificate

### ISSUE THREAD

**Issue by ankitpandey010** [PeculiarVentures/PKI.js#377]
it's not working with ec private key and ec signed certificate in base64 also i tried

**Comment by microshine** [comment]
Please share your example code. PKIjs includes tests for EC algorithms and it works.

**Comment by ankitpandey010** [comment]
yes this is my code i used from your github as a my reference but it's not
working i am using sha 512 and ecdsa and using ec key and ec signed
certificate please help !

thank you

**Comment by ankitpandey010** [comment]
> Please share your example code. PKIjs includes tests for EC algorithms and it works.



> Please share your example code. PKIjs includes tests for EC algorithms and it works.

yes you can find my file name as app.component.ts in my repo in the cv project and I am trying to create a pkcs12 or pfx file using a EC private key and EC certificate and using sha-512 algorithm and putting in your code please resolve this issue thank you!
